pkg/image/registry/imagestream/strategy.go

```diff
@@ -487,6 +488,79 @@ func (s Strategy) Decorate(obj runtime.Object) error {
 	return nil
 }
 
+// Export prepares the object for exporting.
+func (s Strategy) Export(obj runtime.Object, exact bool) error {
```

Comment:
Too many lines duplicated from https://github.com/openshift/origin/pull/8570/files/a0cb78e55d74cda0a332475f9c53f261d811939d#diff-45acca70c84566c7c58b97519d5caf48R153. Can this method  be invoked from the defaultExporter?